### DISCUSSION TITLE
Using Orchard Core Modules

### DISCUSSION THREAD

**Discussion by nikbakhts** [OrchardCMS/OrchardCore#6929] [Q&A] [answered]
Good day Orchard Community,

I'm trying to create a modular MVC application using OC Framework by following the documents and videos I found through OC docs site however still there seems to be a step (or multiple steps) I am missing.

It works fine when I create my custom module and add it to my application. However, still I couldn't figure out how to use OC modules. Here is an example when I'm trying to use OrchardCore.Users in my modular application.

1- I followed this document to create a modular application using OC Framework: https://docs.orchardcore.net/en/dev/docs/guides/create-modular-application-mvc/

2- Then I added OrchardCore.Users module to my application (installing its NuGet package).

Now, when I try to access the login page (https://localhost:44300/Account/Login), I get an error:

Microsoft.AspNetCore.Authentication.Cookies.CookieAuthenticationHandler: Information: Identity.Application was not authenticated. Failure message: Unprotect ticket failed

What I am I missing here?

**Comment by ns8482e** [comment, marked as answer]
@nikbakhts `OrchardCore.Users` is module for `Orchard Core CMS` and doesn't work as standalone in `Orchard Core Framework`

**Reply by nikbakhts** [reply]
Thanks Niraj
